src/codeu/chat/client/commandline/Chat.java

```diff
@@ -402,15 +453,28 @@ public void invoke(List<String> args) {
                             message != null;
                             message = message.next()) {
           System.out.println();
-          System.out.format("USER : %s\n", message.message.author);
+          System.out.format("USER : %s\n", findUsername(message.message.author));
           System.out.format("SENT : %s\n", message.message.creation);
           System.out.println();
           System.out.println(message.message.content);
           System.out.println();
         }
         System.out.println("---  end of conversation  ---");
       }
+      
+      // Find the user with the given UUID and return their username.
+      // If no user has the given UUID this will return "Anonymous".
+      private String findUsername(Uuid author) {
+        for (final UserContext user : context.allUsers()) {
```

Comment:
Update: I didn't realize that `findUser` is searching by name and `findUsername` is searching by Uuid.  For now, I'd be OK with leaving them the way they are.